File: Python/1064.py
# -*- coding: utf-8 -*-
val = 0
valPo = 0
cont = 0

for x in range(0, 6):
    val = float(input())
        
    if val > 0:
        valPo += val
        cont += 1
        
print(cont , "valores positivos")
print("%.1f" % (valPo / cont))

File: Java/1064.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        double val[] = new double[6];
        double valPo = 0;
        int cont = 0;
        for (int i = 0; i < 6; i++) {
            val[i] = leitor.nextDouble();
        }
        
        for (int i = 0; i < 6; i++) {
            if (val[i] > 0) {
                valPo += val[i];
                cont++;
            }
        }
        
        System.out.println(cont + " valores positivos");
        System.out.printf("%.1f\n", (valPo / cont));
    }
 
}